packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -219,11 +241,14 @@ function HeaderBar(props) {
 			</ul>
 			<ul className={classNames(theme['tc-header-bar-actions'], theme.right)}>
 				{props.search && <Components.Search renderers={renderers} {...props.search} />}
-				{props.help && <Components.Help renderers={renderers} {...props.help} t={props.t} />}
-				{props.user && <Components.User renderers={renderers} {...props.user} />}
 				{props.notification && (
 					<Components.AppNotification renderers={renderers} {...props.notification} t={props.t} />
 				)}
+				{props.help && <Components.Help renderers={renderers} {...props.help} t={props.t} />}
+				{!props.user && (
+					<Components.Information renderers={renderers} {...props.information} t={props.t} />
```

Comment:
props.information seems required so you have two choices:
* make it optional (add && props.information)
* add a default value to empty object

Reply:
I don't get your second statement : if I provide an empty object to information, the test will be useless